src/openvic-simulation/map/Map.hpp

```diff
@@ -14,7 +14,10 @@ namespace OpenVic {
 	struct Mapmode : HasIdentifier {
 		friend struct Map;
 
-		using colour_func_t = std::function<colour_t(Map const&, Province const&)>;
+		/* Bottom 32 bits are the base colour, top 32 are the stripe colour, both in ARGB format with the alpha channels
+		 * controlling interpolation with the terrain colour (0 = all terrain, 255 = all corresponding RGB) */
+		using base_stripe_t = uint64_t;
```

Comment:
I'm unsure about the name `base_stripe_t` however it'll have to work for now, I don't have a better name on hand, perhaps it should just be `colour_t`

Reply:
Yeah it was the best I could come up with, IMO colour_t would cause confusion with the single colour type, especially since it's in the OpenVic namespace so inside Mapmode it would be ambiguous which one it refers to.